api/health/health_test.go

```diff
@@ -213,7 +213,7 @@ func TestPassingThenFailingChecks(t *testing.T) {
 	h.Start(context.Background(), checkFreq)
 	defer h.Stop()
 
-	awaitReadiness(h)
+	awaitReadiness(h, true)
 	awaitHealthy(h, true)
 	awaitLiveness(h, true)
```

Comment:
I'm ok keeping these since they're already here but we could just use `require.Eventually` for this

Reply:
done